files/default/solr-import-backup.sh

```diff
@@ -10,8 +11,8 @@ fi
 SOLR_DIR=/var/solr/data/$CORE_NAME/data
 ARCHIVE_NAME=$1
 
-sudo systemctl stop solr
+systemctl stop solr
 sudo -u solr rm -f $SOLR_DIR/index.properties
```

Comment:
you removed sudo from the command calls but left it in for the swap users. You should be able to use ``su - solr ``

Reply:
Using `su` would be possible, yes, but what is the advantage over `sudo`?